jerry-main/main-unix.c

```diff
@@ -513,25 +573,28 @@ main (int argc,
 
   ret_value = jerry_create_undefined ();
 
-  for (int i = 0; i < exec_snapshots_count; i++)
+  if (jerry_is_feature_enabled (JERRY_FEATURE_SNAPSHOT_EXEC))
   {
-    size_t snapshot_size;
-    const uint8_t *snapshot_p = read_file (exec_snapshot_file_names[i], &snapshot_size);
-
-    if (snapshot_p == NULL)
+    for (int i = 0; i < exec_snapshots_count; i++)
     {
-      ret_value = jerry_create_error (JERRY_ERROR_COMMON, (jerry_char_t *) "");
-    }
-    else
-    {
-      ret_value = jerry_exec_snapshot ((void *) snapshot_p,
-                                       snapshot_size,
-                                       true);
-    }
+      size_t snapshot_size;
+      const uint8_t *snapshot_p = read_file (exec_snapshot_file_names[i], &snapshot_size);
 
-    if (jerry_value_has_error_flag (ret_value))
-    {
-      break;
+      if (snapshot_p == NULL)
+      {
+        ret_value = jerry_create_error (JERRY_ERROR_COMMON, (jerry_char_t *) "");
```

Comment:
I've wrote a descriptive information of the 'message' property of this error `ret_value` object.